homeassistant/components/ecobee/config_flow.py

```diff
@@ -0,0 +1,173 @@
+"""Config flow to configure ecobee."""
+import voluptuous as vol
+from copy import copy
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import callback
+
+from .const import (
+    CONF_HOLD_TEMP,
+    CONF_REFRESH_TOKEN,
+    DATA_ECOBEE_CONFIG,
+    DOMAIN,
+    _LOGGER,
+)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
```

Comment:
Please remove the decorator. It's not needed when we pass the domain as keyword argument.